lib/rules/space-infix-ops.js

```diff
@@ -4,6 +4,98 @@
  */
 "use strict";
 
+/**
+ * Returns the first token which violates the rule
+ * @param {SourceCode} sourceCode The source code object to get the operator
+ * @param {ASTNode} left - The left node of the main node
+ * @param {ASTNode} right - The right node of the main node
+ * @param {string} op - The operator of the main node
+ * @returns {Object} The violator token or null
+ * @private
+ */
+function getFirstNonSpacedToken({ sourceCode, left, right, op }) {
```

Comment:
It's changed to named parameters because having too many unnamed parameters is confusing.
i.e. keep forgetting which one is the first and which one is the second...